randovania/gui/ui_files/preset_prime_patches.ui

```diff
@@ -179,6 +179,20 @@
                 </property>
                </widget>
               </item>
+              <item>
+               <widget class="QCheckBox" name="blue_save_doors_check">
+                <property name="text">
+                 <string>Unlock Save Station Doors</string>
+                </property>
+               </widget>
+              </item>
+              <item>
+               <widget class="QLabel" name="blue_save_doors_label">
+                <property name="text">
+                 <string>Ensure all Save Station doors are blue doors, even with door lock rando enabled</string>
```

Comment:
I would say "force" instead of "ensure" since this can be applied to non door-lock seeds, and there are ice save doors in mines